src/Application/Fees/Calculator.php

```diff
@@ -21,19 +21,21 @@ class Calculator
 
     private const string STRIPE_FEE_MAIN_PERCENTAGE_STANDARD = '1.5';
 
+    /** @var string[]   Whole currency unit (e.g. pounds) fee charged *by us* for Stripe credit/debit
+     *                  card donations. These values were chosen based on a Stripe support email about
+     *                  their own core fees in mid 2021 BUT they don't necessarily reflect what Stripe
+     *                  charge *us* due to special contract arrangements.
+     */
     private const array STRIPE_FEES_FIXED = [
```

Comment:
Not a blocker as the other changes are good, but maybe worth making things clearer still by renaming this? e.g. from `STRIPE_FEES_FIXED` to just `FEES_FIXED` to further avoid confusion between fee charged to us and fee charged by us. Probably goes for the two other `STRIPE` constants - the point of the calculator is all about deciding what we charge, not what we pay.

Reply:
Makes sense, done in https://github.com/thebiggive/matchbot/pull/876/commits/2ee4436a529d434dafb919704d2fb716a8eccf9e